### PR TITLE
New LDAP attribute schacPersonalUniqueCode

### PR DESCRIPTION
**Pull request by balcirakpeter**
- new attribute deffinition was added to the perun ldap schema.
- attribute was reflected also in modules and ldapc component.

### CODE REVIEW

File: perun-utils/ldapc-scripts/schemas/perun-schema.ldif [hunk 1]
```diff
@@ -60,9 +60,10 @@ olcAttributeTypes: {55}( 1.3.6.1.4.1.8057.2.80.61 NAME 'isAssignedWithSubgroups'
 olcAttributeTypes: {56}( 1.3.6.1.4.1.8057.2.80.62 NAME 'tcsMails' DESC 'List of users mail addresses used for TCS' EQUALITY caseIgnoreIA5Match SUBSTR caseIgnoreIA5SubstringsMatch SYNTAX 1.3.6.1.4.1.1466.115.121.1.26{256} )
 olcAttributeTypes: {57}( 1.3.6.1.4.1.8057.2.80.63 NAME 'sshPublicKey' DESC 'User`s public SSH key.' EQUALITY caseExactMatch SYNTAX 1.3.6.1.4.1.1466.115.121.1.15 )
 olcAttributeTypes: {58}( 1.3.6.1.4.1.8057.2.80.64 NAME 'sponsoredMembershipInOrganizations' DESC 'List of sponsored memberships in organizations.' EQUALITY caseExactMatch SUBSTR caseExactSubstringsMatch SYNTAX 1.3.6.1.4.1.1466.115.121.1.15 )
+olcAttributeTypes: {19}( 1.3.6.1.4.1.8057.2.80.23 NAME 'schacPersonalUniqueCodes' DESC 'Unique codes of the user in Perun' EQUALITY caseIgnoreMatch SUBSTR caseIgnoreSubstringsMatch ORDERING caseIgnoreOrderingMatch SYNTAX 1.3.6.1.4.1.1466.115.121.1.15 )
```
Comment (zlamalp) [inline perun-utils/ldapc-scripts/schemas/perun-schema.ldif:63]: Attribute value ordering should be sequential ``{19}`` -> ``{59}`` unless you want to re-order them. But we have another pull-request with added attribute, so it should be probably 60 now.
OID should be also sequentially assigned from CESNET range. Hence ``1.3.6.1.4.1.8057.2.80.65`` or ``....66`` (as we added another attribute).

**Review by zlamalp — CHANGES_REQUESTED** [review]

File: perun-utils/ldapc-scripts/schemas/perun-schema.ldif (hunk 1 above)
Comment (balcirakpeter) [reply perun-utils/ldapc-scripts/schemas/perun-schema.ldif]: copy-paste mistake. Fixed

Comment (balcirakpeter) [reply perun-utils/ldapc-scripts/schemas/perun-schema.ldif]: Fixed

**Review by zlamalp — APPROVED** [review]
I expect this to be merged after #2855, it will need rebase afterwards, but its technically ok now.

**Pull request merged**